content/en/docs/ambient/overview/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Overview
+description: An overview of Istio's ambient dataplane mode.
+weight: 1
+owner: istio/wg-docs-maintainers-english
+test: no
+---
+
+In **ambient mode**, Istio implements its [features](/docs/concepts) using a per-node Layer 4 (L4) proxy, and optionally a per-namespace Layer 7 (L7) proxy. The mesh created when Istio is installed in ambient mode can be referred to as an ambient mesh.
+
+Since workload pods no longer require proxies running in sidecars in order to participate in the mesh, Istio in ambient mode is often informally referred to as "sidecar-less" mesh.
+
+This layered approach allows you to adopt Istio in a more incremental fashion, smoothly transitioning from no mesh, to the secure overlay, to full L7 processing — on a per-namespace basis, as needed. Furthermore, workloads running in different ambient configurations, or with sidecars, interoperate seamlessly, allowing users to mix and match capabilities based on the particular needs as they change over time.
```

Comment:
```suggestion
This layered approach allows you to adopt Istio in a more incremental fashion, smoothly transitioning from no mesh, to a secure L4 overlay, to full L7 processing and policy — on a per-namespace basis, as needed. Furthermore, workloads running in different Istio {{< gloss > }}data plane{{< /gloss > }} modes interoperate seamlessly, allowing users to mix and match capabilities based on the particular needs as they change over time.
```

Reply:
Good call, but loses the nuance you can use L4 only and L4-with-waypoint in the same mesh for different workloads.

I don't think we want to consider "ambient L4" and "ambient L7" as different data plane modes overall, do we?

Reply:
No, I was thinking more "ambient dataplane (mode) with L7 support" 

-> you _always_ have the ambient data plane (mode)
-> you _may_ or _may not_ have L7 support in that data plane (mode) depending on your needs